src/kbc_modules/cc_kbc/crypto/mod.rs

```diff
@@ -17,7 +17,20 @@ const RSA_ALGORITHM: &str = "RSA1_5";
 const RSA_PUBKEY_LENGTH: usize = 2048;
 const NEW_PADDING: fn() -> PaddingScheme = PaddingScheme::new_pkcs1v15_encrypt;
 
-const AES_GCM_256_ALGORITHM: &str = "A256GCM";
+pub const AES_GCM_256_ALGORITHM: &str = "A256GCM";
```

Comment:
while we're at it, I think the correct name should be `AES_256_GCM_ALGORITHM` (similarly the function name `aes_256_gcm_decrypt` and in line 137).